### ISSUE TITLE
no matching manifest for windows/amd64 

### ISSUE THREAD

**Issue by geraldbauer** [docker/for-win#6127]
Following step in https://docs.docker.com/docker-for-windows/
### Expected behavior

> docker run hello-world
> 
> docker : Unable to find image 'hello-world:latest' locally
> latest: Pulling from library/hello-world
> 1b930d010525: Pull complete
> Digest: sha256:c3b4ada4687bbaa170745b3e4dd8ac3f194ca95b2d0518b417fb47e5879d9b5f
> Status: Downloaded newer image for hello-world:latest
> 
> Hello from Docker!
> This message shows that your installation appears to be working correctly.
> ...
> 

### Actual behavior

> docker run hello-world
> 
> Unable to find image 'hello-world:latest' locally
> latest: Pulling from library/hello-world
> C:\Program Files\Docker\Docker\resources\bin\docker.exe: no matching manifest for windows/amd64 10.0.17134 in the manifest list entries.
> See 'C:\Program Files\Docker\Docker\resources\bin\docker.exe run --help'.
> 

### Information
  - Windows Version:10.0.17134 Build 17134
  - Docker Desktop Version: Docker version 19.03.8, build afacb8b
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No
  - Processor:	Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz, 2304 Mhz, 8 Core(s), 16 Logical Processor(s)
  - System Type:	x64-based PC

### Steps to reproduce the behavior

  1. Run > docker run hello-world

**Comment by stephen-turner** [comment]
Thanks for the report. I don't think this is a Docker Desktop problem. It's probably a transient problem in the official images while they are updated. Try again in a bit, and comment here if it's still not working.

**Comment by geraldbauer** [comment]
Thanks. I did a fresh install yesterday and saw this. Tried again today and still see it. Who should I ping to discover how long official images take to be updated?

**Comment by stephen-turner** [comment]
@geraldbauer I just tested on my machine and I'm not seeing any problems. Are you still seeing it? Are you able to pull other images successfully, or are they all broken?

**Comment by geraldbauer** [comment]
Yes I am still seeing this issue. From the system screen I am seeing the system has Intel processors - i9 - but when I run reg query "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion" /v BuildLabEx it shows 17134.1.amd64fre.rs4_release.180410-1804. I think that is why docker is logging  no matching manifest for windows/amd64 10.0.17134 in the manifest list entries.

**Comment by stephen-turner** [comment]
Are you using Linux or Windows containers?

**Comment by geraldbauer** [comment]
Windows containers

**Comment by stephen-turner** [comment]
Do you have the same issue with Linux containers, or is it just Windows containers? Also, is there a reason you chose Windows containers? -- I'm only asking because we often find people who choose Windows containers not understanding the difference when Linux containers would actually be a better option for them.

**Comment by geraldbauer** [comment]
Just Windows containers. Switched to Linux to see if that too experienced any problems. 

Yes, we have to use Windows containers until we have the opportunity to port our legacy code to .NET Core.

**Comment by geraldbauer** [comment]
Oops. Where are my manners. Thank you for looking into this. It is appreciated.

**Comment by stephen-turner** [comment]
@geraldbauer You're welcome.
1) What was the result with Linux containers?
2) Also, did you successfully pull other images in Windows mode, i.e. is it only hello-world that's broken or are all images broken?

Thanks.

**Comment by geraldbauer** [comment]
1. Linux container for hello-world worked as expected.
2. I have was able to pull down the windows image microsoft/dotnet-framework - image id: 59d1b03ee52c.

I was doing baby steps to learn about docker. Thus the lesson on hello-world. I have another machine that hello-world functions on. So my issue is with the one machine that I posted here about. That machine happens to be my work machine and I need to run docker to locally to fix any build issues that arise.

Thank you.
